apps/web/src/containers/TradingContest/points-dashboard/hooks/use-your-points-overview-tables.tsx

```diff
@@ -0,0 +1,185 @@
+import { useTheme } from '@mui/material';
+import { colors } from '@notional-finance/styles';
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+export const useYourPointsOverviewTables = () => {
+  const theme = useTheme();
+  const yourPointsData = [
+    {
+      activeSeason: (
+        <div>
+          Season 1{' '}
+          <span
+            style={{
+              fontSize: '14px',
+              color: colors.greenGrey,
+              fontWeight: '500',
+              marginLeft: theme.spacing(0.5),
+            }}
+          >
+            (April 15 - July 15)
+          </span>
+        </div>
+      ),
+      pointsPerDay: '5,120.32',
+      yourTotalPoints: '10,000.00',
+      pointsReceived: '10,000.00',
```

Comment:
I don't know if the mock is wrong or not but `pointsReceived` => `arbReceived` instead.